terraform/slurm_cluster/examples/slurm_cluster/test_cluster/main.tf

```diff
@@ -20,7 +20,7 @@
 
 module "project_services" {
   source  = "terraform-google-modules/project-factory/google//modules/project_services"
-  version = "~> 12.0"
+  version = "~> 15.0"
```

Comment:
Noting here that 15.0 requires TPG 5.22 and above. 14.5.0 allows 4.x and <=6. I think we should consider that.

Reply:
Based on that, I agree it makes sense to change to 14.5.